test/parallel/test-tls-inception.js

```diff
@@ -29,12 +30,11 @@ const assert = require('assert');
 const tls = require('tls');
 
 const fs = require('fs');
-const path = require('path');
 const net = require('net');
 
 const options = {
-  key: fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem')),
-  cert: fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'))
+  key: fs.readFileSync(fixtures.path('test_key.pem')),
+  cert: fs.readFileSync(fixtures.path('test_cert.pem'))
```

Comment:
this can be:

```js
key: fixtures.readSync('test_key.pem'),
cert: fixtures.readSync('test_cert.pem')
```

Reply:
My apologies for not following through; i pushed those changes, but i see that you have a separate branch already that had the same thing, so we could use either branch to merge.

Reply:
Oh, it is still great to have the correct change in the PR! The way we merge things would result in the same right now.